test/integration/buildruns_to_sa_test.go

```diff
@@ -95,14 +97,24 @@ var _ = Describe("Integration tests BuildRuns and Service-accounts", func() {
 		})
 
 		It("uses the pipeline serviceaccount if exists", func() {
-			Expect(tb.CreateSAFromName("pipeline")).To(BeNil())
+			saName := "pipeline"
+			saCreationError := tb.CreateSAFromName(saName)
+			Expect(saCreationError).To(BeNil())
```

Comment:
nit - call returned errors `err`:

```suggestion
			err = tb.CreateSAFromName(saName)
			Expect(err).To(BeNil())
```

Note that the integration suite has a global error object `err`, which isn't best practice. This should be addressed in a separate PR.

Reply:
fixed, only using `err` now